arm-sql/2014-04-01/swagger/recoverableDatabases.json

```diff
@@ -0,0 +1,381 @@
+            "get": {
+                "tags": [
+                    "GeoBackup"
+                ],
+                "operationId": "Databases_ListGeoBackups",
+                "description": "Returns a list of Azure SQL database geo backups.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/GeoBackupList"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/recoverableDatabases/{databaseName}": {
+            "get": {
+                "tags": [
+                    "GeoBackup"
+                ],
+                "operationId": "Databases_GetGeoBackup",
+                "description": "Returns an Azure SQL Database geo backup.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database to retrieve geo backups for."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/GeoBackup"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/restorableDroppedDatabases": {
+            "get": {
+                "tags": [
+                    "DeletedDatabaseBackup"
+                ],
+                "operationId": "Databases_ListBackupsForDeletedDatabases",
```

Comment:
The public naming for this uses "Deleted".  The powershell cmdlet also uses deleted: Get-AzureRmSqlDeletedDatabaseBackup 

---
In reply to: [80284342](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80284342) [](ancestors = 80284342)